packages/grid/_modules_/grid/hooks/features/rows/useGridEditRows.ts

```diff
@@ -302,11 +310,12 @@ export function useGridEditRows(apiRef: GridApiRef) {
   );
 
   useGridApiEventHandler(apiRef, GRID_CELL_KEYDOWN, handleCellKeyDown);
-  useGridApiEventHandler(apiRef, GRID_CELL_BLUR, handleCellBlur);
   useGridApiEventHandler(apiRef, GRID_CELL_MOUSE_DOWN, preventTextSelection);
   useGridApiEventHandler(apiRef, GRID_CELL_DOUBLE_CLICK, handleDoubleClick);
   useGridApiEventHandler(apiRef, GRID_CELL_EDIT_ENTER, handleEnterEdit);
   useGridApiEventHandler(apiRef, GRID_CELL_EDIT_EXIT, handleExitEdit);
+  useGridApiEventHandler(apiRef, GRID_CELL_FOCUS_OUT, handleClickOutside);
```

Comment:
This is suspicious. `GRID_CELL_FOCUS_OUT` if fire for click away and focus events. I don't get the naming nor the event. Why not?
```suggestion
  useGridApiEventHandler(apiRef, GRID_CELL_CLICK_OUTSIDE, handleClickOutside);
```

Reply:
I updated the handler name. `handleClickOutside` was a inheritance from when I was adding a listener to the `document`. 

`GRID_CELL_CLICK_OUTSIDE` is not good name because the focus out can also be triggered pressing <kbd>Esc</kbd> or programmatically calling `setCellFocus`.